src/exampleTypes/index.js

```diff
@@ -37,41 +42,107 @@ export default ({
   ListGroupItem = 'div',
   Icon,
   Select = DefaultSelect,
+  Loading = StripedLoader,
 } = {}) => {
+  let injectNodeAndLoad = (...args) =>
+    _.flow(
+      Loading,
```

Comment:
@dshishkov We don't have to take `Loading` props now. I think it's cleaner to pass `loaderStyle` to style the loader (only used here: https://github.com/smartprocure/contexture-react/pull/193/files?file-filters%5B%5D=#diff-a4b47316e0d702587a5e046a030ecf21R98)